### ISSUE TITLE
Adding Course Packet to a session auto-adds all packets for that course to the session.

### ISSUE THREAD

**Issue by MindImpMatt** [maltyeva/mindimp-pg#93]
For example, when I added a GP session, and included packets, I didn't just get the packet for that session, but an additional (test packet) that had been created earlier. I should be able to choose which packet(s) are used in a session.

**Comment by maltyeva** [comment]
If it's a test packet, then you should be able to delete it (did i add a delete option?). We didn't talk about this as a feature, but is the option to pick and choose course packets something you would want?

**Comment by jzthompson** [comment]
Yes, I think we should have the option to pick and choose course packets. For example, in the essay course I'll be teaching next week in the orientation program, I'll be putting together specially designed course packets for the compressed version of the course. I wouldn't want these packets to appear in normal course sessions for this course, as they wouldn't fit with the lesson plan and would likely confuse students. Of course, I could just delete these course packets after the orientation course, but there might be instances in the future when we want to have different types of course packets available  for different difficulty levels of the course, or something like that.

**Comment by maltyeva** [comment]
Okay, so thinking about this a little more (and it may be more complicated than we want to make this), we can make each course packet have the option to be used or not, however, would this matter in different concurrently running classes? Like for example, if there are two sessions of Modern Essay running at the same time, would we want a specific course packet to be available for one and not the other?

**Comment by ChasonMindImp** [comment]
That might be useful. If, in the future, we are running multiple sessions
of a course simultaneously, but with different people, or if we continue to
modify so that we have 1, 5, 10-week versions of each class, etc., we'd
want the materials to be specific to the session.

But I see that as a long-term thing, yes?

On Mon, Apr 27, 2015 at 4:59 PM, maltyeva <notifications@github.com> wrote:

> Okay, so thinking about this a little more (and it may be more complicated
> than we want to make this), we can make each course packet have the option
> to be used or not, however, would this matter in different concurrently
> running classes? Like for example, if there are two sessions of Modern
> Essay running at the same time, would we want a specific course packet to
> be available for one and not the other?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/maltyeva/mindimp-pg/issues/93#issuecomment-96573641>.
>



-- 
Chason Dailey | 戴奇森
Program Manager and Advisor* | **思脉 **MindImp*
https://www.mindimp.com/

**Comment by jzthompson** [comment]
Yes, Maria, I think there could be a chance that we'd need different types of course packets for concurrently running sessions of the same class. Examples could be two instructors wishing two teach different books in a lit class, or different course packets for more and less advanced versions of a writing class.

Chason does make a good point that this is a long-term issue. We don't need this functionality this week, and I don't expect we'll have enough students in the second academic term to need it in May or June either.
